src/main/scala/day4/conditions.scala

```diff
@@ -0,0 +1,18 @@
+package day4
+import chisel3._
+import chisel3.util._
+class condition extends Module{

```

Comment:
place a line break between class and import, and capitalize the class and file name.